docs/develop/api-reference/dictionary-collections.md

```diff
@@ -0,0 +1,201 @@
+---
+sidebar_position: 1
+sidebar_label: Dictionary collections
+title: Dictionary collection API reference
+description: Learn how to interact with the API for dictionary collection data types in Momento Serverless Cache.
+slug: /develop/api-reference/collections/dictionary
+---
+
+# Dictionary collection API reference
+
+## Dictionary methods
+This page details the Momento API methods for [dictionary collections](./../datatypes.md#dictionary-collections).
+
+### DictionaryFetch
+
+Gets a dictionary item from a cache.
```

Comment:
Would probably remove the word "item" here as it might be interpreted as only pulling a single field from the dictionary.  Maybe "retrieves an entire dictionary from a cache" or something?